<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckUpdate</class>
 <widget class="QWidget" name="CheckUpdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#CheckUpdate
{
	background-color:white;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>固件更新</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>118</width>
     <height>24</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="startUpdate">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定更新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelUpdate">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>暂不更新</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCheckingNow">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>正在检查更新...</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelUpdateResult">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>发现新版本：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
